packages/bp/src/common/utils.ts

```diff
@@ -17,3 +17,15 @@ export const sanitizeName = (text: string) =>
     .replace(/\s|\t|\n/g, '-')
     .toLowerCase()
     .replace(/[^a-z0-9-_.]/g, '')
+
+export const getErrorMessage = (error: Error | string | unknown): string => {
+  if (typeof error === 'string') {
+    return error
+  }
+
+  if (error instanceof Error) {
+    return error.message
+  }
+
+  return ''
```

Comment:
Wouldn't it be useful to see what was the error "object" anyway? Wrapping it in a `JSON.stringify()` if it doesn't evaluates to one of the `if` statements could be helpful for debugging purposes if we're to reuse this function throughout the project.

Reply:
I agree with you, this is a first pass to set it up. I didn't want to over-engineer it before seeing cases where it needs to be expanded, so I was thinking to improve it when we see a new usage. What do you think?

Reply:
I don't think it's over-engineering but I agree we can go with this and if we see too many empty strings at some point we can just return here and make the change. Thanks!